atom/common/api/atom_bindings.cc

```diff
@@ -175,6 +177,12 @@ v8::Local<v8::Value> AtomBindings::GetProcessMemoryInfo(v8::Isolate* isolate) {
   return dict.GetHandle();
 }
 
+// static
+int64_t AtomBindings::GetCreationTime(v8::Isolate* isolate) {
+  auto metrics = base::CurrentProcessInfo::CreationTime();
+  return metrics.ToInternalValue();
```

Comment:
this is deprecated

Reply:
Should probably use

```cpp
base::CurrentProcessInfo::CreationTime().ToJsTime()
```

Refs: [`ToJsTime()`](https://cs.chromium.org/chromium/src/base/time/time.h?dr=CSs&g=0&l=571)

Then users can just do `new Date()` on the ret value to get a JS date

Reply:
Bonus points for a new `mate` converter to `v8::Date`

https://v8docs.nodesource.com/node-0.8/d4/df7/classv8_1_1_date.html